sdk/eventhub/eventhubs-checkpointstore-blob/README.md

```diff
@@ -1,19 +1,105 @@
-# Azure Event Hubs checkpoint blob client library for Javascript
+# Azure Event Hubs Checkpoint Store library for Javascript using Storage Blobs
 
-This template serves as a starting point for JavaScript libraries targeting both Node and the Browser and implemented in TypeScript.
+An Azure Blob storage based solution to store checkpoints and to aid in load balancing when using `EventProcessor` from the [@azure/event-hubs](https://www.npmjs.com/package/@azure/event-hubs) library
 
-[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/eventhubs-checkpointstore-blob) | [Package (npm)](https://www.npmjs.com/package/@azure/eventhubs-checkpointstore-blob) | [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/eventhubs-checkpointstore-blob/index.html) | [Product documentation](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-event-processor-host) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/eventhubs-checkpointstore-blob/samples)
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/eventhubs-checkpointstore-blob) | [Package (npm)](https://www.npmjs.com/package/@azure/eventhubs-checkpointstore-blob) | [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/eventhubs-checkpointstore-blob/index.html) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/eventhubs-checkpointstore-blob/samples)
 
 ## Getting started
 
+### Install the package
+
+Install the Azure Event Hubs Checkpoint Store Blob library using npm
+
+`npm install @azure/eventhubs-checkpointstore-blob@latest`
+
+**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/) and a
+[Event Hubs Namespace](https://docs.microsoft.com/en-us/azure/event-hubs/) to use this package, and a [Storage account](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+If you are using this package in a Node.js application, then use Node.js 8.x or higher.
+
+### Configure Typescript
+
+TypeScript users need to have Node type definitions installed:
+
+```bash
+npm install @types/node
+```
+
+You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your tsconfig.json. Note that if you have enabled `compilerOptions.esModuleInterop`, `allowSyntheticDefaultImports` is enabled by default. See [TypeScript's compiler options handbook](https://www.typescriptlang.org/docs/handbook/compiler-options.html) for more information.
+
 ## Key concepts
 
 ## Examples
```

Comment:
```suggestion
 ## Examples

### Create a Partition Manager using Azure Blob Storage

To create a Partition Manager that uses Azure Blob Storage, you need to first have an [Azure subscription](https://azure.microsoft.com/free/) and an [Azure Storage account](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)

Then, you can use the below code snippet to create a Partition Manager

```javascript
import { ContainerClient } from "@azure/storage-blob",
import { BlobPartitionManager } from "@azure/eventhubs-checkpointstore-blob"

const containerClient = new ContainerClient("storage-connection-string", "container-name");
await containerClient.create(); // This can be skipped if the container already exists

const partitionManager =  new BlobPartitionManager(containerClient);
```